azure-devops/global/abstracts/script-e2e-tests.yml

```diff
@@ -0,0 +1,91 @@
+parameters:
+  - name: TAG_NAME
+    type: string
+  - name: PLUGIN_NAME
+    type: string
+  - name: OS_PORT
```

Comment:
Ok, think a little bit about it... I don't want to stress you too much, but if we were building a generic pipeline for all Node apps, to run e2e testing, it would be nicer if you have a way of setting an array of ports, instead of hardcoding the parameters here. What do you think?